src/components/cylc/analysis/BoxPlot.vue

```diff
@@ -97,6 +98,75 @@ export default {
     },
   },
 
+  setup (props) {
+    const reducedAnimation = useReducedAnimation()
+
+    const chartOptions = computed(() => ({
```

Comment:
@cylc/data-workflows 

Heads up, this is just a different way of defining a `computed` property which is more kosher with the use of "composables" (aka things that start with the prefix `use`). No further changes required.